src/main/res/values/strings.xml

```diff
@@ -1181,13 +1181,13 @@
 
 	<!-- 2020-02-20 -->
 	<string name="upgrade_message">Updated to version %s</string>
-	<string name="error_toast_failed_open_external_browser">Failed to open url \"%s\" in external browser</string>
-	<string name="error_toast_no_email_apps">No email apps installed!</string>
-	<string name="bug_chooser_title">Email bug report</string>
+	<string name="error_toast_failed_open_external_browser">Could not open the URL "%s" in any web-browser</string>
+	<string name="error_toast_no_email_apps">No e-mail apps installed.</string>
+	<string name="bug_chooser_title">E-mail bug report</string>
 
 	<!-- 2020-06-08 -->
-	<string name="subscription_successful">Successfully subscribed to %s</string>
-	<string name="unsubscription_successful">Successfully unsubscribed from %s</string>
+	<string name="subscription_successful">Subscribed to %s</string>
+	<string name="unsubscription_successful">Unsubscribed from %s</string>
```

Comment:
Preferred original

Reply:
The word "successfully" is at best redundant. If not, it suggests this often fails in a way that unsubscribes, but not successfully so. It is also longer, and software doesn't do this. This is debug style info in the UI, which isn't good.

Reply:
Okay, let's go with the new version